packages/repository/test/acceptance/has-many.relation.acceptance.ts

```diff
@@ -59,9 +59,6 @@ describe('HasMany relation', () => {
     expect(persisted.toObject()).to.deepEqual(order.toObject());
   });
 
-  // This should be enforced by the database to avoid race conditions
-  it('reject create request when the customer does not exist');
```

Comment:
What about `it.skip`?

Reply:
`mocha` shows tests skipped with `it.skip` as `pending` still. I'm ok to abandon this PR if everyone is ok with it but I found the pending test to be annoying. 

A `.skip` will be better so anyone looking for the culprit can find it easily at the very least.